addons/position_manager/README.md

```diff
@@ -0,0 +1,24 @@
+# Position Manager
+
+Set and save screen position per-character.  
+
+Command: //pm set <Charactername|all> pos_x pos_y
```

Comment:
I would probably add backticks around this for code formatting, but just a minor suggestion.

Reply:
Fixed.